source/Ship.cpp

```diff
@@ -1920,7 +1920,15 @@ void Ship::Move(vector<Visual> &visuals, list<shared_ptr<Flotsam>> &flotsam)
 				acceleration += angle.Unit() * thrust / mass;
 
 				// Only create the afterburner effects if the ship is in the player's system.
-				isUsingAfterburner = !forget;
+				if(!forget && !Attributes().AfterburnerEffects().empty())
+					for(const EnginePoint &point : enginePoints)
+					{
+						// Stream the afterburner effects outward in the direction the engines are facing.
+						Point effectVelocity = -6. * angle.Unit();
```

Comment:
Is this an effect that should be attached to the ship that generates it? Or is this an effect similar to "engine exhaust" where it actually makes a lot of sense not to attach it to the ship?